lib/vty.c

```diff
@@ -3673,6 +3673,7 @@ static int vty_mgmt_get_data_result_notified(
 	return 0;
 }
 
+#if 0
```

Comment:
This is going to get hit by checkpatch.pl, it doesn't like #if 0'd code.

Reply:
Let's please ignore this error for now. I don't want to spend time moving this anywhere. We have other `#if 0` code in the codebase, so it's not something strictly prohibited.

My overall idea is to start returning errors in this JSON/XML format in `mgmt_msg_error` message. I think this makes sense, because user requests a response in a specific format and we're returning a simple string in case of an error. This code may be reused for this, so I don't want to delete it completely.

Reply:
OK I think you can trick checkpatch.pl then to pass by using a symbolic constant that i undefined

`#ifdef DONT_DEFINE_UNTIL_NEEDED`